Core/Config.cpp

```diff
@@ -1301,6 +1301,15 @@ void Config::SetAppendedConfigIni(const Path &path) {
 	appendedConfigFileName_ = path;
 }
 
+void Config::updateAfterSettingAutoFrameSkip() {
```

Comment:
nit: Should be a capital U to follow other methods.

That said, maybe this could be combined with `PostLoadCleanup()` instead (it probably should have the `iFrameSkip = 1` bit anyway), and we could just run that when needed.  Better to be more generic in methods exposed by Config, I'd say.

-[Unknown]